karton/core/task.py

```diff
@@ -301,10 +313,20 @@ def unserialize(
 
         :meta private:
         """
+
+        def unserialize_resources(value):
```

Comment:
```suggestion
        def unserialize_resources(value: Any) -> Any:
```

Or should we use generics?